website/app/components/doc/a11y-support.hbs

```diff
@@ -5,4 +5,4 @@
 
 <h2 class="doc-text-h2 doc-page-sidecar-scroll-margin-top" id="support">Support</h2>
 <p class="doc-text-body">If any accessibility issues have been found within this component, let us know by
-  <a class="doc-link-generic" href="https://github.com/hashicorp/design-system/issues/new/choose">submitting an issue</a>.</p>
\ No newline at end of file
+  <a class="doc-link-generic" href="https://go.hashi.co/hds-support">submitting an issue</a>.</p>
```

Comment:
send straight to the intake form or send them to the support page first?

Reply:
I'm usually inclined to route folks directly to their destination and not require an additional click, but given this may prompt more external feedback than other areas, could we route this one to the Support page and add a section there for external folks that links to the GitHub issues page?

Reply:
> this may prompt more external feedback than other areas

It hasn't once so far, but I see your point. This goes a bit to the larger question about when to route directly the form and when to route to the support page 🤔